aepsych/database/tables.py

```diff
@@ -73,15 +75,15 @@ class DBMasterTable(Base):
     children_raw = relationship("DbRawTable", back_populates="parent")
 
     @classmethod
-    def from_sqlite(cls, row):
+    def from_sqlite(cls, row: Dict[str, Any]) -> 'DBMasterTable':
```

Comment:
Please double check that search ahead types works without an extra import in all versions of Python we support (3.10-3.12).

Reply:
Yes, I double-checked and it is supported in 3.7 and above.